tests/test_pdf.py

```diff
@@ -817,8 +817,7 @@ def test_model_integration_fixed_parameters():
         'parameters': [{'name': 'mypoi', 'inits': [1], 'fixed': True}],
     }
     model = pyhf.Model(spec, poi_name='mypoi')
-    assert model.config.suggested_fixed() == [False, True]
-    assert model.config.poi_index == 1
+    assert model.config.suggested_fixed()[model.config.par_slice('mypoi')] == [True]
```

Comment:
the order changed again didn't it? why not just swap it around instead of combining the two? We won't catch a bug if both parameters are somehow "fixed" moving forward (so we lose coverage)